sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 5.0.0-beta.5 (Unreleased)
-
-### Features Added
+## 5.0.0-beta.5 (2021-07-07)
 
 ### Breaking Changes
-
-### Key Bugs Fixed
-
-### Fixed
-
+- Renamed `ServiceBusEntityType` property to `EntityType`.
+- Renamed `messageActions` and `sessionActions` parameters to `actions` in `MessageProcessor` and `SessionMessageProcessor`.
+- Renamed `MaxBatchSize` to `MaxMessageBatchSize` in `ServiceBusOptions`.
```

Comment:
@pakrym, @jsquire do we want to rename the equivalent in EH to `MaxEventBatchSize`?

Reply:
I wasn't in the board reviews, so please take my thoughts with a grain of salt... but I think that makes sense.

Reply:
This was just based on a comment in the API review from @KrzysztofCwalina

Reply:
I think it makes sense.